bindings-python/gci/componentmodel.py

```diff
@@ -14,6 +14,7 @@
 class SchemaVersion(enum.Enum):
     V1 = 'v1'
     V2 = 'v2'
+    V2_1 = 'v2.1'
```

Comment:
should this rather become v3?

Reply:
The only difference is that v2.0 will add the version as identity. Or is there any other difference

If that is the only difference then I think v3 does suggest a more drastic change of the structure so v2.1 seems good to me.

Reply:
@schrodit otoh, we did not actually use semantic versioning.. so my argument against `2.1` (and in favour of `(v)3` would be that this would make it a bit easier to sort (since we would then simply use integers). unfortunately, the version is a `str`, rather than an `int` :-/

Reply:
+ the next version _will_ be incompatible, which would - also after semver-semantics - mandate a new major version.

Reply:
why incompatible?

Reply:
we no longer implicitly inject `version` to the extraIdentity, but identities must be unique. so w/o change, some component-descriptors will become invalid after changing to v2+

Reply:
I will rm it again from this PR, as it does not belong here, anyway